release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+
+[[ocp-37-installation]]
+=== Installation
+
+[[ocp-37-migrate-etcd-before-upgarde]]
+==== Migrate etcd Before OpenShift Container Platform 3.7 Upgrade
+Starting in {product-title} 3.7, the use of the etcd3 v3 data model is required.
+
+{product-title} gains performance improvements with the v3 data model. In order
+to upgrade the data model, an embedded etcd configuration option in no longer
+allowed. Embedded is not co-located and mainly used in single-master
+deployments. Migration scripts will convert the v3 data model and allow you to
+move an embedded etcd to an external etcd either on the same host or a different
+host than the masters. In addition, there is a new scale up ability for etcd
+clusters.
+
+See
+xref:../install_config/upgrading/migrating_embedded_etcd.adoc#install-config-upgrading-etcd-data-migration[Migrating
+Embedded etcd to External etcd] for more information.
+
+[[ocp-37-modular-installer]]
+==== Modular Installer to Allow Playbooks to Run Independently
+
+The installer has been enhanced to allow administrators to install specific
+components. By breaking up the roles and playbooks, there is better targeting of
+ad hoc administration tasks.
+
+[[new-install-experience-around-phases]]
+==== New Installation Experience Around Phases
+When you run the installer, {product-title} now reports back at the end what
+phases you have gone through.
+
+If the installation fails during a phase, you will be notified on the screen
+along with the errors from the Ansible run. Once you resolve the issue, rather
+than run the entire installation over again, you can pick up from the failed
+phase. This results in an increased level of control during installations and
+results in time savings.
+
+[[ocp-37-increased-control-over-image-stream-templates]]
+==== Increased Control Over Image Stream and Templates
+With {product-title} 3.7, there is added control over whether or not your cluster
+automatically upgrades all the content provided during cluster upgrades.
+
+Edit the `openshift_install_examples` variable in the hosted file or set it as a variable in the installer.
+
+----
+RPM = /etc/origin/examples /etc/origin/hosted
+Container = /usr/share/openshift/examples /usr/share/openshift/hosted
+
+openshift_install_examples=false
+----
+
+Setting `openshift_install_examples` to `false` will cause the installer to not
+upgrade the imagestream and templates. `True` is the default behavior.
+
+[[ocp-37-install-config-cfme-from-ocp-installer]]
+==== Installation and Configuration of CFME 4.6 from the OpenShift Installer
+
+CloudForms Management Engine (CFME) 4.6 is now fully supported running on
+{product-title} 3.7 as a set of containers. CFME is an available API endpoint on
+all {product-title} clusters that choose to use it. More cluster administrators
+are now able to leverage CFME and begin experiencing the insight and automations
+available to them in {product-title}.
+
+To install CFME 4.6:
```

Comment:
Since CFME deployment has a relatively stringent requirements of OCP cluster, and also has many customizing template parameters, I think we could add a link here to direct readers to the "deploying cfme" doc, should be the address http://file.rdu.redhat.com/~adellape/110117/cfme37/install_config/cfme/index.html going to be published.